starter-code/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="../bootstrap-4.1.0/css/bootstrap.css">
+  <link rel="stylesheet" href="./styles.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+</head>
+
+<body>
+
+  <div class="container-fluid">
+    <div class="row top-navigation-bar">
+      <div class="col-sm-4">
+        <h1>REVERA</h1>
+      </div>
+      <div class="col-sm-6">
+        <ul class="nav">
+          <li class="nav-item">
+            <a class="nav-link active" href="#">Home</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link active" href="#">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link active" href="#">Sample Page</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link active" href="#">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link active" href="#">Blog</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link active" href="#">Parent Page</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link active" href="#">Content</a>
+          </li>
+        </ul>
+      </div>
+      <div class="col-sm-2"></div>
+    </div>
+  </div>
+  <div class="row hero-photo">
+  </div>
+
+  <div class="container-fluid row-under-carousel">
+    <div class="row">
+      <div class="container">
+        <div class="row inner-row">
+          <div class="col-md-4">
+            <h3>Web Design</h3>
```

Comment:
to get the images needed you can check out glyphicons from the bootstrap website and for the circle you can just add a border and control the radius